webapp/src/blocks.tsx

```diff
@@ -211,7 +211,9 @@ export class Editor extends srceditor.Editor {
                 scaleSpeed: 1.05
             },
             rtl: Util.userLanguageRtl()
-        });
+        };
+        let blocklyOptions = $.extend({}, defaultOptions, pxt.appTarget.appTheme.blocklyOptions || {});
```

Comment:
we have our own function to merge JSON. Let's try to jquery creep in the codebase.